eng/testing/xunit/xunit.console.targets

```diff
@@ -13,8 +13,8 @@
 
   <ItemGroup>
     <!-- Configures xunit to not print out passing tests with output when diagnostic messages are enabled. -->
-    <SetScriptCommands Condition="'$(TargetOS)' == 'windows'" Include="set XUNIT_HIDE_PASSING_OUTPUT_DIAGNOSTICS=1" />
-    <SetScriptCommands Condition="'$(TargetOS)' != 'windows'" Include="export XUNIT_HIDE_PASSING_OUTPUT_DIAGNOSTICS=1" />
+    <SetScriptCommands Condition="'$(OS)' != 'Windows_NT'" Include="set XUNIT_HIDE_PASSING_OUTPUT_DIAGNOSTICS=1" />
+    <SetScriptCommands Condition="'$(OS)' == 'Windows_NT'" Include="export XUNIT_HIDE_PASSING_OUTPUT_DIAGNOSTICS=1" />
```

Comment:
"fix XUNIT_HIDE_PASSING_OUTPUT_DIAGNOSTICS for windows" means that `TargetOS` was empty so we never triggered  this line? I am wondering why we changed from `TargetOS` to `OS`.

Reply:
The generated script was wrong and complained to log on linux helix.

This is the change that introduced it https://github.com/dotnet/runtime/pull/105392